src/display/api.js

```diff
@@ -775,16 +805,16 @@ class PDFDocumentProxy {
   }
 
   /**
-   * @type {Object} A subset of the current {DocumentInitParameters}, which are
-   *   either needed in the viewer and/or whose default values may be affected
-   *   by the `apiCompatibilityParams`.
+   * @type {DocumentInitParameters} A subset of the current
+   *   {DocumentInitParameters}, which are either needed in the viewer and/or
+   *   whose default values may be affected by the `apiCompatibilityParams`.
    */
   get loadingParams() {
     return this._transport.loadingParams;
   }
 
   /**
-   * @type {PDFDocumentLoadingTask} The loadingTask for the current document.
+   * @type {PDFDocumentLoadingTasks} The loadingTask for the current document.
```

Comment:
The class is named `PDFDocumentLoadingTask` (and that shouldn't be changed); why did you add a trailing `s` here?